05-how-to-contribute-to-open-source.md

```diff
@@ -144,10 +144,12 @@ You'll need to have these tools donwloaded and installed on your local machine:
 3. Clone your forked repository to your computer.
 4. Run `npm install` to install the dependencies.
 5. Create a new branch and use a descriptive name related to your contribution — for example, `feat/add-alice`.
-6. Run `npm run contributors: add` in your terminal.
+6. Run `npm run contributors:add` in your terminal.
 
    Follow the instructions to add yourself to the guestbook. After you finish and click enter, you must click enter again to confirm your choices.
 
+   ![screenshot of a terminal displaying the steps to creating a profile for the guestbook](./images/cli-tool.gif)
```

Comment:
When writing an alt text, we don't need to mention "image" or "picture" or similar term. 😊
You can read this [accessibility article](https://accessibility.huit.harvard.edu/describe-content-images) for your reference.

```suggestion
   ![add contributor with CLI on a terminal GIF](./images/cli-tool.gif)
```